easy_my_coop_initial_import/wizards/pay_capital_release_invoices/pay_capital_release_invoices.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+from odoo import fields, models
+
+from odoo.addons.queue_job.job import job
+
+log = logging.getLogger(__name__)
+
+
+class PayCapitalReleaseInvoices(models.TransientModel):
+    _name = "pay.capital.release.invoices"
+
+    journal_id = fields.Many2one(
+        "account.journal",
+        string="Payment Journal",
+        required=True,
+        domain=[("type", "in", ("bank", "cash"))],
+    )
+
+    def pay(self):
+        self.with_delay()._create_payments()
+
+    @job
+    def _create_payments(self):
+        log.info("Preparing data to create payments...")
+        capital_release_invoices = (
+            self.env["account.invoice"]
+            .search(
+                [
+                    ("state", "=", "open"),
+                    ("subscription_request", "!=", None),
+                    (
+                        "subscription_request.migrated_cooperator_register_number",
+                        "!=",
+                        None,
+                    ),
+                ]
+            )
+            .sorted(
+                lambda i: i.subscription_request.migrated_cooperator_register_number
+            )
+        )
+
+        self.AccountRegisterPaymentWizard = self.env["account.register.payments"]
+
+        self.account_payment_method = self.env.ref(
+            "account.account_payment_method_manual_in"
+        )
+
+        log.info("Starting to create payments")
+        for cs_invoice in capital_release_invoices:
+            log.info(
+                "Creating payment for invoice {}".format(cs_invoice.invoice_number)
+            )
+            wizard = self.AccountRegisterPaymentWizard.with_context(
+                {"active_model": "account.invoice", "active_ids": [cs_invoice.id]}
+            ).create(
+                {
+                    "payment_date": cs_invoice.subscription_request.capital_release_request_date,  # noqa
+                    "payment_method_id": self.account_payment_method.id,
+                    "journal_id": self.journal_id.id,
+                }
+            )
+            try:
+                wizard.create_payments()
+                self.env.cr.commit()
+            except Exception as error:
+                self.env.cr.commit()
```

Comment:
I think `commit` should be removed here since you already committed the previous payments on the last loop. You're committing a transient model (which is ok) but you might also commit partial data created in `create_payments` before the error was raised.